### ISSUE TITLE
Double Curly Brackets not showing up in body text between <code></code> tags

### ISSUE THREAD

**Issue by Sharlikran** [jekyll/jekyll#7965]
## My Environment
Using Windows 7, Ruby, Jekyll, and other Gems

I am unsure of where to get version info in Windows. I just installed Jekyll so I would presume it's fairly recent.
## Expected Behaviour
```HTML
== Syntax =====================================================================
=== Your Pages Title

Your page title will become the file line on the page and will be assighnd as the first header. To indicate this text use the following Wrye Tag.
. When you add <code>{{PAGETITLE="Your Page Title"}}</code> with double <code>{{</code><code>}}</code>s
```
It just occurred to me that Jekyll probably parses my HTML page looking for things in curly brackets. When it finds them then it omits them. 

## Current Behavior

The issue seems to be that the builder thinks it is liquid and treats it as invalid syntax. It's not liquid it's my own style of Markdown that I process with a Python script.

![image](https://user-images.githubusercontent.com/1912426/71962252-ec31a080-31ad-11ea-94be-dc20c4185cd8.png)
![image](https://user-images.githubusercontent.com/1912426/71962301-fce21680-31ad-11ea-9686-80a85aeb7972.png)

What can I put so that the markdown tags are not treated as Liquid tags?

## Code Sample
```Text
. When you add <code>{{PAGETITLE="Your Page Title"}}</code> with double <code>{{</code><code>}}</code>s
```

**Comment by Sharlikran** [comment]
I thought I would try `{% %}` and that outright crashes the builder
![image](https://user-images.githubusercontent.com/1912426/71962920-66aef000-31af-11ea-86c4-b8126c4b2cf9.png)

**Comment by Sharlikran** [comment]
![image](https://user-images.githubusercontent.com/1912426/71963122-d02efe80-31af-11ea-816b-ba0ef28ddb3f.png)
`When you add <code>{<PAGETITLE="Your Page Title">}</code> with double curly braces <code>{</code><code>}</code>s`

I was fairly disapointed Icould use teh syntax I wanted, so I thought I would try `{< Tag >}` Apparently that's taken as well.

**Comment by ashmaroli** [comment]
@Sharlikran This is not a bug but rather just a beginner's mistake.
Liquid parses your contents and converts all instances `{{ ... }}` and `{% ... %}` and renders them irrespective of whether you wrap them in `<pre/>`, `<code/>` or not.

To instruct Liquid to leave a instance of `{{ ... }}` and `{% ... %}` unrendered, wrap them in `{% raw %} {% endraw %}` Liquid tags:
```html
<code>{% raw %}{{ page.title }}{% endraw %}</code>
```

**Comment by Sharlikran** [comment]
@ashmaroli 
I won't be able to do that. I can't even `re.sub()` that out.

For Example:

`{% raw %}{{PAGETITLE="Contributing"}}{% endraw %}`

I use `re.match()` which matches the beginning of the line. So with those tags it may display on the webpage correctly but now the regex will fail.

If I were to use `re.sub(r'{% raw %}', '', line) to remove it and the trailing `{% endraw %}` then I will be back where I started with just 

![image](https://user-images.githubusercontent.com/1912426/71963634-e9847a80-31b0-11ea-8bf1-e934dec573dc.png)

I am parsing a text file using Python and I just happened to use double curly brackets for the syntax. These tags are not liquid syntax of any kind.

Is there no ither way to tell Jekyll to ignore them?

If I was writing the HTML myself I would have this:

```HTML
<p>When you add <code>{{<PAGETITLE="Your Page Title">}}</code> with double curly braces
```

This is only an issue because Jekyll is treating a non liquid tag as a liquid tag.

**Comment by ashmaroli** [comment]
Its not that Jekyll is treating non-Liquid as Liquid. This is how Liquid itself works.
I don't know why you want Python to parse your Jekyll site's source-code.
If you just want Jekyll to render a HTML file with python code in a pre-formatted block, you need not worry about `raw endraw` ending up in the final HTML generated.

**Comment by Sharlikran** [comment]
I am being brief as to not cause confusion. I am explaining exactly what I am doing. You are presuming that I am parsing my sites web source rather then using Jekyll.

I am simply writing instructions on how to do something. Like steps on how to program in C++, Python or, how to use Markdown. If I put double curly brackets `{{}}` or `{%%}` or `{<>}` then Jekyll can not handle it. It isn't Liquid.

I understand that if I were explaining how to write C++ source, `{% raw %}` and `{% endraw %}` would work for that. However, this is my own custom markdown language I am writing from scratch. I don't want people to have to put in their text file `{% raw %}` and `{% endraw %}` just so that Jekyll ignores it.

I will see if I can approach it a different way. Thank you for your time.

**Comment by Sharlikran** [comment]
I hate to be wrong and it can be embracing, but I am still partially not to blame.

The cause of my confusion is because I was not omitting the superfluous markdown tags that are being treated as liquid. In my defense it is because they are being hidden when I view the page.

So as I forgot to omit `{{CONTENTS=2}}` or `{{PAGETITLE}}` those were being added to array to later be converted to HTML. Once I viewed the page they became hidden. So I didn't know they were there.

I only noticed it once I tried to find a different combination for the beginning and ending of the tag.

![image](https://user-images.githubusercontent.com/1912426/71969082-d7a7d500-31ba-11ea-95ee-2d8b68a5a37d.png)

At this point I am simply writing superfluous information to the HTML file and I can deal with that now that I can see it.

I still feel there should be some something else that happens when Jekyll encounters an invalid tag other then state it is invalid and crash out or output `ECONNABORTED` as shown in my screen shots.

Both errors were from my custom markdown being treated as Liquid.
